functions/go/set-namespace/README.md

```diff
@@ -50,23 +53,27 @@ In the following example, the `set-namespace` function will update:
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: service-account
-  namespace: original-namespace
+  name: sa
+  namespace: ns
+  annotations:
+    config.kubernetes.io/depends-on: /namespaces/ns/ServiceAccount/foo # <= this will NOT be updated (resource not declared)
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   ...
+  annotations:
+    config.kubernetes.io/depends-on: /namespaces/ns/ServiceAccount/sa # <== this will be updated (resource declared)
 subjects:
   - kind: ServiceAccount
     name: default # <================== name default is used
-    namespace: original-namespace # <== this will be updated
+    namespace: ns # <================== this will be updated
   - kind: ServiceAccount
-    name: service-account # <========== name matches above ServiceAccount
-    namespace: original-namespace # <== this will be updated
+    name: sa # <======================= name matches above ServiceAccount
+    namespace: ns # <================== this will be updated
   - kind: ServiceAccount
     name: other-service-account # <==== this will NOT be updated
```

Comment:
> # <==== this will NOT be updated

This comment actually belongs to the next line, right?

Reply:
Looks like you are correct here - I will fix this up